main.py

```diff
@@ -42,6 +42,12 @@ def main():
 
 	if defaults.getboolean("UseExampleRenderPasses"):
 		config.read_string(fix_config("config/example.renderpasses.ini"))
+	elif defaults.get("RenderPassFile"):		
+		if os.path.isfile(defaults.get("RenderPassFile")):
```

Comment:
Store the RenderPassFile into a variable instead of getting it again from config

Reply:
Done.